### ISSUE TITLE
I want to bind html as vue-component

### ISSUE THREAD

**Issue by kunJuly** [avil13/vue-sweetalert2#59]
like this
this.$swal({
          type: 'success',
          target: document.getElementsByClassName('live-left')[0],
          // text: 'hellllllll',
          html: `<PopupWindow></PopupWindow>`
        })

`PopupWindow` is my component

Can I do that?thank~!

**Comment by panudetjt** [comment]
I want a solution to this problem either.

**Comment by fahmizulhasymi** [comment]
After some digging, i found [related post ](https://css-tricks.com/creating-vue-js-component-instances-programmatically/) to the stackoverflow link from @panudetjt 

for my case, i need to create interactive Vue Form. and i use this package so much in my apps. 

This is what i came up with.
[live sample here](https://codesandbox.io/s/vue-sweetalert-inside-component-n4l80)

i create component Instances programmatically, and mount it everytime i need the form (in case there is some preprocess). and get form result before i destroy it.

ref: 
[v-model on component](https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components)

**Comment by LeonAlvarez** [comment]
> After some digging, i found [related post ](https://css-tricks.com/creating-vue-js-component-instances-programmatically/) to the stackoverflow link from @panudetjt
> 
> for my case, i need to create interactive Vue Form. and i use this package so much in my apps.
> 
> This is what i came up with.
> [live sample here](https://codesandbox.io/s/vue-sweetalert-inside-component-n4l80)
> 
> i create component Instances programmatically, and mount it everytime i need the form (in case there is some preprocess). and get form result before i destroy it.
> 
> ref:
> [v-model on component](https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components)

The problem with that solution is that it will be another Vue instance so, if you are using for example vuex you won't have access to it there

**Comment by fahmizulhasymi** [comment]
yes, that is why i use function initForm and getResult as the bridge.
